#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-flac.dpatch by Joachim Reichel <joachim.reichel@gmx.de>
##
## DP: fix typo w.r.t. flac support

@DPATCH@
diff -urNad normalize-audio-0.7.7~/src/normalize-mp3.in normalize-audio-0.7.7/src/normalize-mp3.in
--- normalize-audio-0.7.7~/src/normalize-mp3.in	2005-09-12 07:31:15.000000000 +0200
+++ normalize-audio-0.7.7/src/normalize-mp3.in	2007-01-19 21:37:37.000000000 +0100
@@ -294,7 +294,7 @@
     $path = find_prog("flac");
     if ($path) {
 	$path .= " -s -d -o %w %m";
-	$FLAC = $path;
+	$FLACDECODE = $path;
     }
 }
 
@@ -303,7 +303,7 @@
     $path = find_prog("flac");
     if ($path) {
 	$path .= " -s -o %m %w";
-	$FLAC = $path;
+	$FLACENCODE = $path;
     }
 }
 
